multi-magit.el

```diff
@@ -705,7 +705,7 @@ like to select some using `multi-magit-list-repositories'? ")
 
 ;;;; Magit-status Sections
 
-(defvar magit-multi-magit-repo-section-map
+(defvar multi-magit-magit-repo-section-map
```

Comment:
This name is not arbitrary. `magit` expects a variable named like this in `(magit-insert-section (multi-magit-repo ...) ...)`.

Reply:
Oh, wops! (Magic names are perhaps worth a code comment)

Reply:
Agreed. Pushed a comment.